examples/README.md

```diff
@@ -260,3 +268,29 @@ Loaded key to 0x80000001
 ```
 
 The `keyload` tool takes only one argument, the filename of the stored key. Because the information what is key scheme (RSA or ECC) is contained within the key blob.
+
+## Storing keys into the TPM's NVRAM
+
+These examples demonstrates how to use the TPM as secure vault for keys. There are two programs, one to store a TPM key into the TPM's NVRAM and another to extract the key from the TPM's NVRAM. Both examples can use parameter encryption to protect from MITM attacks. The Non-volatile memory location is protected with a password authorization that is passed in encrypted form, when "-aes" or "-xor" is given on the commmand line.
+
+Before running the examples, make sure there is a keyblob.bin generated using the keygen tool. The key can be of any type, RSA, ECC or symmetric. The example will store the private and public part. In case of a symmetric key the public part is meta data from the TPM. How to generate a key you can see above, in the description of the keygen example.
+
+Typical output for storing and then reading an RSA key using parameter encryption:
+
+```
+$ ./examples/nvram/store -aes
```

Comment:
Can we add support for specifying the key type? Or are you waiting for this to merge so you can do that as part of PR #143 ?

Reply:
Good idea, we could generate RSA or ECC. I would address this in 143 and we can have the parameter encryption fix merged here.

Reply:
@dgarske RSA ECC or Symmetric key all work with nvram/store,

because nvram/store uses readKeyBlob

and whatever is written in the KeyBlob->pub/priv.size that is then stored into the NVRAM (size-wise).

Did you mean to be able to store only the private material (-priv option perhaps)?

Could be useful for symmetric keys. Also -pub in cases when only the pub part has to be stored in NVRAM.